README.md

```diff
@@ -26,269 +24,224 @@ pip install jupyterhub-ltiauthenticator
 
-    You should enter the following information into the appropriate component
-    settings:
+#### LTI 1.1 Configuration XML Settings
 
-    **LTI ID**: The value you entered for `your-hub-name` in step 4.
+The LTI 1.1 configuration XML settings are available at `/lti11/config` endpoint. Some LMS vendors accept XML and/or URLs that render the configuration XML to simplify the LTI 1.1 External Tool installation process.
 
-    **LTI URL**: Should be set to `your-hub-url/hub/lti/launch`. So if your hub
-    is accessible at `http://datahub.berkeley.edu`, **LTI URL** should be
-    `http://datahub.berkeley.edu/hub/lti/launch`
+You may customize these settings with the `config_*` configuration options described in the [common configuration settings](#common-configuration-settings) section.
 
-    **LTI Launch Target**: Should be set to **New Window**.
+#### Custom Configuration with JupyterHub's Helm Chart
 
-    **Custom parameters**: The only currently supported custom parameter is
-    `next`, which can contain the relative URL that the user should be redirected
-    to after authentication. For example, if you are using
-    [nbgitpuller](https://github.com/data-8/nbgitpuller) and want the user to see
-    [this file](https://github.com/binder-examples/requirements/blob/HEAD/index.ipynb) after
-    logging in, you could set the **Custom parameters** field to the following
-    string:
+If you are running **JupyterHub within a Kubernetes Cluster**, deployed using helm, you need to supply the client key & shared secret with the `lti.consumers` key. The example below also demonstrates how customize the `lti.username_key` to set the user's email as the JupyterHub username and the `lti.config_icon` to define a custom external tool icon when using the LTI 1.1 configuration XML endpoint:
 
-    ```js
-    [
-      "next=/hub/user-redirect/git-pull?repo=https://github.com/binder-examples/requirements&subPath=index.ipynb",
-    ];
-    ```
-
-    _Note_: If you have a `base_url` set in your jupyterhub configuration, that should be prefixed to your next parameter. ([Further explanation](#common-gotchas))
-
-8.  You are done! You can click the Link to see what the user workflow would look
-    like. You can repeat step 6 in all the units that should have a link to the
-    Hub for the user.
-
-## Canvas
-
-The setup for Canvas is very similar to the process for edX.
+```yaml
+# Custom config for JupyterHub's helm chart
+auth:
+  type: "lti"
+  lti:
+    consumers: { "client-key": "client-secret" }
+    username_key: "lis_person_contact_email_primary"
+    config_icon: "https://my.static.assets/img/icon.jpg"
```

Comment:
The documentation above is outdated, it should be like this since ~0.10.X or 0.11.X i think, and the chart is now at 1.1.3.

```suggestion
# Custom config for JupyterHub's helm chart
hub:
  config:
    # Additional documentation related to authentication and authorization available at
    # https://zero-to-jupyterhub.readthedocs.io/en/latest/administrator/authentication.html
    JupyterHub:
      authenticator_class: lti
    LTI11Authenticator:
      consumers: { "client-key": "client-secret" }
      username_key: "lis_person_contact_email_primary"
      config_icon: "https://my.static.assets/img/icon.jpg"
```

Reply:
Corrected.